src/img/icons/caret-up-small.svg

```diff
@@ -1 +1,1217 @@
-<svg width="10" height="6" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)"><path fill-rule="evenodd" clip-rule="evenodd" d="M9.65 5.06c.1-.1.15-.21.1-.26L4.97.01c-.017-.017-.048-.02-.086-.01-.035-.005-.065 0-.084.02l-.083.083A.544.544 0 0 0 4.71.11l-.006.006L.02 4.81c-.04.04 0 .16.1.26l.35.34c.1.1.21.15.26.1l4.15-4.16 4.16 4.16c.04.04.16 0 .26-.1l.35-.35z" fill="#3C444D" fill-opacity=".7"/></g><defs><clipPath id="clip0"><path transform="rotate(90 9.77 0)" fill="#fff" d="M9.77 0h5.53v9.77H9.77z"/></clipPath></defs></svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I changed them because the old versions were not consistent and not how our current SVGs are.

The down caret's viewbox was 27x27, and the icon itself didn't even cover the whole viewbox, only like a third of it. This made it so that when we need to use it we have to inflate the width and height of the icon in order to actually get the size that we want the icon to be.

An example of this was the caret used in the `Select` core component, the caret there needed to have a size of around 7px so in order to get that size right with the old SVG, the container needed to be 21x21 in order to compensate for the empty space in the SVG itself, which I feel like is inconsistent and cumbersome to deal with.
![Screenshot from 2022-06-22 13-45-23-1](https://user-images.githubusercontent.com/18473896/175093561-bcfedf7b-3594-4c68-90be-3922b2b882bd.png)

With the new version, you can expect to assign the icon or its container `7px` for the width and the icon to actually be `7px`. It also complies with our rule of having a viewbox of 30x30:
![Screenshot from 2022-06-22 13-46-34](https://user-images.githubusercontent.com/18473896/175095273-7913d901-8d0a-4aa9-8bf8-67ae40944258.png)

The up version of the caret had a viewbox of 10x6. It was also not centered in the SVG:
![Screenshot from 2022-06-22 13-45-49](https://user-images.githubusercontent.com/18473896/175093610-a97c3d7b-9883-40ad-97be-627e8dfe7fec.png)

The new version has a viewbox of 30x30 as per the law, it's centered and has the same color as the down version which is some extra consistency points:
![Screenshot from 2022-06-22 13-47-01](https://user-images.githubusercontent.com/18473896/175097956-284b5c77-1c8b-4321-a809-ffa655377ef4.png)
 
I'm not a SVG expert tho, can you explain to me what would make those a better, more optimized SVG? I see some props that could be shaved off but I don't think that's all that you are referring to, right?

Reply:
I agree it should be updated, although these SVGs include every SVG from the icon pack in them. I will clean it up and send you the right ones.

For reference:
![Screenshot 2022-06-22 203538](https://user-images.githubusercontent.com/33682027/175111457-f85bc424-8a36-4a85-be2d-a584e83105d8.png)

You can open SVGs in an IDE to see what is in them.

Reply:
What the fuck? I was using Inkscape, but I'll try it with the IDE :thinking: Thank you very much for the info Arren!